core/controllers/profile_test.py

```diff
@@ -1153,7 +1153,11 @@ def test_put_function(self) -> None:
         self.logout()
 
     def test_email_provider_error(self) -> None:
-        def raise_exception() -> None:
+        # Here we use type Any because we will not be using the
```

Comment:
I think, for clarity, you'll want to call this something like mock_add_user_to_raise_exception() and define its method signature as "unused_email, unused_tag, name=None". This is because the mock method that you're swapping in should mimic the original method.

I think this should address the 405 issues you're getting.

Reply:
Done